eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -18,12 +18,20 @@ steps:
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
         --identity "$(Build.QueuedBy)"
-      $resolvedIdentity = $result[-1] | ConvertFrom-Json
 
-      Write-Host $resolvedIdentity
+      $resolvedIdentity = ""
```

Comment:
Something to consider. Instead of doing a bunch of extra processing in inline scripting we could add an option to the identity-resolver tool to simply output the username in the devops comment format and so we don't have to parse write and parse json and instead just have the specific variable set or not set.

Reply:
Just to be clear DevOps simply parses all standard out so the identity-resolver could also output the magic vso string and DevOps would pick it up.